app/models/batch_context.rb

```diff
@@ -15,7 +15,7 @@ class BatchContext < ApplicationRecord
 
   belongs_to :user
   has_many :job_runs, dependent: :destroy
-  has_one :globus_destinations, dependent: :destroy
+  has_one :globus_destination, dependent: :destroy
```

Comment:
I think it makes more sense for this to be singular?

Reply:
Yeah, that was my bad, thanks for fixing. I think Rails still did the right thing(?), but should be singular.